conu/apidefs/file.py

```diff
@@ -0,0 +1,84 @@
+"""
+Abstract class definitions for files in containers.
+"""
+
+from conu.apidefs.container import Container
+from conu.apidefs.image import Image
+
+
+class File(object):
```

Comment:
our idea was to implement this interface: https://docs.python.org/3/glossary.html#term-file-object

that we would have methods on Container and Image classes which would return real objects which implement read(), write(), close()...